### PR TITLE
LPS-200168

### PR DESCRIPTION
**Pull request by MicaelleSilvaa**
Ticket: https://liferay.atlassian.net/browse/LPS-200168

### CODE REVIEW

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -302,6 +302,14 @@
 		],
 		"to": "serveImage(PortalUtil.getHttpServletRequest(param#0#), PortalUtil.getHttpServletResponse(param#1#))"
 	},
+	{
+		"classNames": [
+			"AssetCategoryLocalService",
+			"AssetCategoryLocalServiceUtil"
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:308]: @MicaelleSilvaa, can we also include here the `AssetCategoryService` and `AssetCategoryServiceUtil`?

If these classes exist in `master` and also have the methods mentioned in `from` and `to` then it makes sense we include them. From now on, if it's applicable, we should always use, for these specific cases, the following classes:

- `*LocalService`
- `*LocalServiceUtil`
- `*Service`
- `*ServiceUtil`

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json (hunk 1 above)
Comment (natocesarrego) [reply modules/util/source-formatter/src/main/resources/dependencies/replacements.json:308]: @MicaelleSilvaa, do you have any update about this comment?

Thanks

**Comment by MicaelleSilvaa** [comment]
ci:test:sf

**Comment by MicaelleSilvaa** [comment]
ci:test:hotel

**Comment by natocesarrego** [comment]
Any update regarding my [previous comment](https://github.com/natocesarrego/liferay-portal/pull/1407#discussion_r1386926142), @MicaelleSilvaa?

Thanks

**Comment by natocesarrego** [comment]
Please fix conflicts and resend

**Pull request closed without merging**